internal/http/services/owncloud/ocs/handlers/apps/sharing/shares/pending.go

```diff
@@ -166,15 +173,15 @@ func (h *Handler) RejectReceivedShare(w http.ResponseWriter, r *http.Request) {
 		Share: &collaboration.Share{
 			Id: &collaboration.ShareId{OpaqueId: shareID},
 		},
-		State:  collaboration.ShareState_SHARE_STATE_REJECTED,
-		Hidden: h.getReceivedShareHideFlagFromShareID(r.Context(), shareID),
+		State: collaboration.ShareState_SHARE_STATE_REJECTED,
 	}
-	updateMask := &fieldmaskpb.FieldMask{Paths: []string{"state", "hidden"}}
+	updateMask := &fieldmaskpb.FieldMask{Paths: []string{"state"}}
```

Comment:
we need to make sure the share managers respect the field mask!

Reply:
jsonsc3 handles it, memory handles it, owncloudsql handles it (but does not know anything about the hidden flag), json handles it (but does not know anything about the hidden flag), cs3 also has it. good to merge